projects/tuvali.md

```diff
@@ -0,0 +1,93 @@
+# Project Name
+Tuvali - Means a tunnel in [Tamil](https://en.wikipedia.org/wiki/Tamil_language)
+
+# Project Description
+This library contains the implementation for Android & IOS for the Openid4BLE specification. It's written in Kotlin & Swift.
+
+# Alignment with the OpenWallet Foundation Mission
+This library implements the BLE-based local transfer of Verifiable Presentation (VP). An offline-based mode provides us with the ability to present a VP in many crises and crowded situations, from stadium entries to social benefits in remote areas. This is an important building block for future digital identity wallets to be inclusive. Since it is written in Kotlin & Swift, it can be used widely.
+
+# Current Code of Conduct
+[Code of Conduct](https://docs.mosip.io/1.2.0/community/code-of-conduct)
+
+We will also follow the LF Code of Conduct
+
+# TAC Sponsor
+No one Now
+
+# Project License
+Currently, the project is hosted with MPL 2.0 license. We will move to Apache license 2.0 upon approval of the project.
+
+# Source Control
+[Repository](https://github.com/mosip/tuvali)
+
+# Issue Tracker
+[Jira](https://mosip.atlassian.net/issues/?jql=labels%20%3D%20%22BLE%22)
+
+# External Dependencies
+```
+  implementation "com.facebook.react:react-native:+"
+  implementation "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"
+  implementation 'org.jetbrains.kotlinx:kotlinx-coroutines-core:1.3.2'
+
+  implementation 'org.bouncycastle:bcpkix-jdk15to18:1.68'
+  implementation 'org.bouncycastle:bcprov-jdk15to18:1.68'
+  implementation 'com.github.snksoft:crc:1.1.0'
+  implementation "org.jetbrains.kotlin:kotlin-reflect:1.7.0"
+```
+npm dependencies
+
+```
+"@arkweid/lefthook": "^0.7.7",
+    "@commitlint/config-conventional": "^17.0.2",
+    "@react-native-community/eslint-config": "^3.0.2",
+    "@release-it/conventional-changelog": "^5.0.0",
+    "@types/jest": "^28.1.2",
+    "@types/react": "~17.0.21",
+    "@types/react-native": "0.70.0",
+    "commitlint": "^17.0.2",
+    "del-cli": "^5.0.0",
+    "eslint": "^8.4.1",
+    "eslint-config-prettier": "^8.5.0",
+    "eslint-plugin-prettier": "^4.0.0",
+    "jest": "^28.1.1",
+    "pod-install": "^0.1.0",
+    "prettier": "^2.0.5",
+    "react": "18.1.0",
+    "react-native": "0.70.6",
+    "react-native-builder-bob": "^0.20.0",
+    "release-it": "^15.0.0",
+    "typescript": "^4.5.2"
+```
+
+# Release Methodology
+[Release Process](https://docs.mosip.io/1.2.0/community/release-process)
+
+# Initial Maintainers
```

Comment:
Can you comment on the diversity of maintainers? Specifically, are all three maintainers part of the MOSIP organization?

Reply:
Yes as of now.